autoscheduler/scraper/README.md

```diff
@@ -0,0 +1,11 @@
+# Scraper
+
+## About
+
+This will consists of two main parts:
+
+1) All of the scraping logic for retrieving data from Howdy/compassxe and loading it into the database
+    - In part, this includes all of the main models for the project, such as Course, Section, or Instructor.
+2) The view logic for retrieving these models from the database, such as:
+    - Retrieving all of the sections for `CSCE-221`.
+    - Retrieving the departments that start with "C"
```

Comment:
Same as for scheduler, this will most likely be redundant once we actually start working on the project